lib/clearance/password_strategies/bcrypt_migration_from_sha1.rb

```diff
@@ -22,10 +29,12 @@ def initialize(user)
       end
 
       def authenticated?(password)
+        warn "#{Kernel.caller.first}: #{DEPRECATION_MESSAGE}"
```

Comment:
To elaborate - if I use a heredoc, I think I have to leaves the messages left aligned in the source or I'll actually get those spaces in the terminal output. Is that correct?